### ISSUE TITLE
Broken functionality between: 4.9 and 6.3: tableBuilding callback

### ISSUE THREAD

**Issue by KES777** [olifolkerd/tabulator#4626]
**Describe the bug**
https://tabulator.info/docs/6.3/lifecycle#initialization

**Tabulator Info**
- 6.3

**Working Example**
```
<div id="customized-table"></div>
<script>
new Tabulator( '#customized-table', {
  tableBuilding: function(){ alert( 'sdfa' ) },
});
</script>
```

In previous versions I could write:
```
let opts =  Tabulator.defaultOptions;
opts.tableBuilding  = initLookups;
```

And all my tables receive some initialization. But now in all places in the code I need to write:
```
let t = new Tabolator(...);
t.on("tableBuilding", initLookups);
```
which is not convenient.

**To Reproduce**
1. Run the example
2. tableBuilding callback is not fired.

**Expected behavior**
I should see alert.

**Screenshots**
![image](https://github.com/user-attachments/assets/95cad993-5d19-4ab8-afc9-5e8c7f37694e)

**Desktop (please complete the following information):**
 - OS: Linux Mint 21.3 
 - Browser FF
 - Version 130

**Comment by olifolkerd** [comment]
Hey @KES777 

Looks like you are making a big version jump there, there have been a huge number of changes to the library since version 4.9

As you will see in the upgrade guide, there were substantial changes in the 5.0 release that include a move away from callbacks, to using a new event system.

Full documentation on this can be found here:
https://tabulator.info/docs/5.0/upgrade

Cheers

Oli :)

**Comment by KES777** [comment]
Yes, I switched my code to `t.on( "tableBuilding", callback );`.

But, as you can see, the documentation for 6.3 still documents that it calls callback, it does not fire the new event.

From your response it is not clear will you fix that part of documentation or not.

Thanks.

**UPD**
Ah, I reread that part couple of times more, now I understand what happened.
> The `tableBuilding` callback is triggered

This makes me think that the regular callback passed as option to the construction is fired. But here you mean that the **callback** argument passed to `t.on( "tableBuilding", callback );` call is triggered.

It would be nice to have a link from this section to https://tabulator.info/docs/6.3/events#table.
In this case it will be more clear about callback.
